modules/opencl/src/image/layercl.cpp

```diff
@@ -132,7 +132,7 @@ void LayerCL::setDimensions(size2_t dimensions) {
     if (dimensions == dimensions_) {
         return;
     }
-
+    dimensions_ = dimensions;
     cl::Image2D* resizedLayer2D = new cl::Image2D(OpenCL::getPtr()->getContext(), CL_MEM_READ_WRITE,
```

Comment:
This should be a `make_unique`, not `new cl::Image` ;)

Reply:
fixed it along with an annoying error